api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -320,6 +323,27 @@ private boolean isComplianceTrainingCompliant(DbUser user) {
     return ctTimes.isBypassed() || (ctTimes.isComplete() && !ctTimes.hasExpired());
   }
 
+  // these 2 modules will not be checked at all until we flip the feature flag
+
+  private boolean isProfileConfirmationCompliant(DbUser user) {
+    if (!configProvider.get().access.enableAccessRenewal) {
+      return true;
+    }
+
+    final ModuleTimes profileTimes = new ModuleTimes(user.getProfileLastConfirmedTime(), null);
+    return profileTimes.isComplete() && !profileTimes.hasExpired();
+  }
+
+  private boolean isPublicationConfirmationCompliant(DbUser user) {
+    if (!configProvider.get().access.enableAccessRenewal) {
+      return true;
+    }
+
+    final ModuleTimes publicationTimes =
+        new ModuleTimes(user.getPublicationsLastConfirmedTime(), null);
```

Comment:
nit: maybe create a second constructor for this case, or alternatively inline comment like `/* bypassTime */ null`